boot/zephyr/Kconfig

```diff
@@ -320,6 +307,28 @@ config BOOT_SWAP_SAVE_ENCTLV
 
 endif # !SINGLE_APPLICATION_SLOT
 
+config SINGLE_APPLICATION_SLOT_RAM_LOAD
+	bool "RAM load for single application slot"
+	select SINGLE_APPLICATION_SLOT
```

Comment:
```suggestion
```

This needs to be a distinct mode, it can use the same .c file and check the ifdefs but this Kconfig needs to be its own Kconfig and not do anything with other MCUboot mode Kconfigs